api/src/main/java/org/openmrs/module/mksreports/MKSReportsActivator.java

```diff
@@ -53,23 +47,6 @@ public void started() {
 		adminSrvc.setGlobalProperty("htmlformentryui.customPrintPageName", MKSReportsConstants.PATIENTHISTORY_ID);
 		adminSrvc.setGlobalProperty("htmlformentryui.customPrintTarget", "_blank");
 		
-		if (StringUtils.isBlank(adminSrvc.getGlobalProperty("mksreports.brandingLogo"))) {
```

Comment:
This was replaced with the GP in config.xml for defaultValue and new code in report renderer https://github.com/mekomsolutions/openmrs-module-mksreports/blob/c073e4c712eedd7304539be10dc273d8249a7e7a/api/src/main/java/org/openmrs/module/mksreports/renderer/PatientHistoryXmlReportRenderer.java#L197-L221